CONTRIBUTING.md

```diff
@@ -88,6 +88,45 @@ The devfile samples used in this devfile registry are stored in the `extraDevfil
 4) Fill in the fields in the angle brackets based on your sample. Note that there must be only one git remote for the devfile sample.
 5) Open a pull request against this repository with your changes.
 
+### Adding a new version
+
+In case you want to add another version to a new devfile sample you can update the existing sample inside the `extraDevfileEntries.yaml` file:
+
+1) Verify your Sample functions with Che.
+  
+    - Opening the URL `https://workspaces.openshift.com/#<repository_url>` in your browser should start a workspace where `exec` commands run successfully. Type `task+<space>` in the IDE command palette to see the list of available commands.
```

Comment:
What does this test?

Reply:
@maysunfaisal Checks if your sample work. I've tried this URL https://github.com/thepetk/devfile-sample-go-basic/tree/update_devfile in order to check the sample and build it in workspaces.

Maybe I should add more explicit info about branches?